src/Types.jl

```diff
@@ -257,43 +257,46 @@ mutable struct EnvCache
     # registered package info:
     uuids::Dict{String,Vector{UUID}}
     paths::Dict{UUID,Vector{String}}
+    names::Dict{UUID,Vector{String}}
```

Comment:
It's a bit unfortunate to store things like this where each `Vector{String}` is in the vast majority of elements in `names` only one element.

Storing it something like

```
uuids_collection::Vector{UUID}
uuids::Dict{String, UnitRange{Int}}
```

and extracting `vew`s from that might be better.